File: Gamestore-ui/src/app/core/models/publisher.ts
export interface Publisher {
  id: string;
  companyName: string;
  homePage?: string | null;
  description?: string | null;
}

export interface PublisherUpsertRequest {
  publisher: {
    id?: string;
    companyName: string;
    homePage?: string | null;
    description?: string | null;
  };
}

File: GameStore/Models/Publisher.cs
namespace GameStore.Models;

public class Publisher
{
    public Guid Id { get; set; }

    public required string CompanyName { get; set; }

    public string? HomePage { get; set; }

    public string? Description { get; set; }

    public ICollection<Game> Games { get; set; } = [];
}
